test/lib/absinthe/relay/connection_test.exs

```diff
@@ -573,6 +574,15 @@ defmodule Absinthe.Relay.ConnectionTest do
     end
   end
 
+  describe "when defined with non_null: true" do
+    test "it will define edges and edge as non_null" do
+      always_present_pets_edge =
+        CustomConnectionAndEdgeFieldsSchema.__absinthe_type__(:always_present_pets_edge)
+
+      assert
```

Comment:
@jeroenvisser101 I created a PR based on your comment, this here is causing it to fail.

Reply:
Yeah, that's right, the tests were still pending, the plan was to do assertions on the type to check that it had `non_null` added to it. Any hints on how to do this?

There are also 3 possible ways of defining non_null (separately with non_null_edge, non_null_edges and non_null, all of which should be tested).